articles/aks/rdp.md

```diff
@@ -23,128 +23,63 @@ This article assumes that you have an existing AKS cluster with a Windows Server
 
 You also need the Azure CLI version 2.0.61 or later installed and configured. Run `az --version` to find the version. If you need to install or upgrade, see [Install Azure CLI][install-azure-cli].
 
-## Deploy a virtual machine to the same subnet as your cluster
+## Establish a jump server to your AKS cluster
 
-The Windows Server nodes of your AKS cluster don't have externally accessible IP addresses. To make an RDP connection, you can deploy a virtual machine with a publicly accessible IP address to the same subnet as your Windows Server nodes.
+The Windows Server nodes of your AKS cluster don't have externally accessible IP addresses. To make an RDP connection, you can deploy an SSH container on a Linux node to establish a jump server to enble RDP.
 
 The following example creates a virtual machine named *myVM* in the *myResourceGroup* resource group.
```

Comment:
Remove `The following example creates a virtual machine named *myVM* in the *myResourceGroup* resource group`?